src/store/store.ts

```diff
@@ -49,8 +52,14 @@ const rootReducer = combineReducers({
     snackbar: snackbarReducer,
 });
 
-
 export function configureStore(history: History) {
+    const projectPanelMiddleware = dataExplorerMiddleware(
```

Comment:
What are the pros of moving construction of projectPanel and favoritePanel middlewares into this place?